tests/contest/contest/src/tests/process_user/process_user_test.rs

```diff
@@ -0,0 +1,49 @@
+use anyhow::{Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, Spec, SpecBuilder, UserBuilder};
+use rand::Rng;
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+use crate::utils::test_inside_container;
+
+// Generates a Vec<u32> with a random number of elements (between 5 and 15),
+// where each element is a random u32 value between 0 and 65535.
+fn generate_random_vec() -> Vec<u32> {
+    let mut rng = rand::thread_rng();
+    let vec_size = rng.gen_range(5..=15);
+    (0..vec_size).map(|_| rng.gen_range(0..=65535)).collect()
```

Comment:
```suggestion
    let vec_size = rng.gen_range(5..=10);
    (0..vec_size).map(|_| rng.gen_range(100..=200)).collect()
```

Reply:
I fixed this.

https://github.com/youki-dev/youki/pull/2978/commits/49a229baea7330657bfd11f5a86d2adc814ebb5c